README.md

```diff
@@ -24,13 +24,15 @@ Download and install node. Go to https://nodejs.org/en/download/ and follow the
 
 Once you have Docker and Node installed and setup then do the following:
 
-    copy server/src/configs/sample.config.ts -> server/src/configs/config.ts
+    cp server/sample-bluebutton-config.json -> server/.bluebutton-config.json
```

Comment:
```suggestion
    cp server/sample-bluebutton-config.json  server/.bluebutton-config.json
```
Suggested change so that a copy & paste to the command line works.